pyqir-jit/src/intrinsics.rs

```diff
@@ -21,13 +21,24 @@ fn get_current_gate_processor() -> ForceSomeRwLockWriteGuard<'static, BaseProfil
     v
 }
 
+/// # Safety
+///
+/// This function should not be called directly. It is intended to be
+/// called by QIR applications during JIT execution.
 #[no_mangle]
 pub unsafe extern "C" fn __quantum__qis__h__body(qubit: QUBIT) {
     log::debug!("/__quantum__qis__h__body/");
     let mut gs = get_current_gate_processor();
     gs.h(qubit);
 }
 
+/// # Safety
+///
+/// This function should not be called directly. It is intended to be
+/// called by QIR applications during JIT execution.
+/// # Panics
```

Comment:
Nit, but Markdown headers should be separated by at least one preceding blank line.